packages/react/src/components/Menu/Menu.tsx

```diff
@@ -88,6 +88,12 @@ interface MenuProps extends React.HTMLAttributes<HTMLUListElement> {
   /**
    * Specify a DOM node where the Menu should be rendered in. Defaults to document.body.
    */
+  menuTarget?: Element;
```

Comment:
Since this prop is already within the Menu component it feels redundant to call it `menuTarget` instead of `target`

Reply:
Thank you for the suggestion. I have changed **menuTarget** prop back to **target** for **Menu** component.